persistence/javadsl/src/test/scala/com/lightbend/lagom/javadsl/persistence/multinode/AbstractClusteredPersistentEntitySpec.scala

```diff
@@ -38,9 +38,16 @@ abstract class AbstractClusteredPersistentEntityConfig extends MultiNodeConfig {
       lagom.persistence.read-side.run-on-role = "read-side"
       terminate-system-after-member-removed = 60s
 
-      # increase default barrier-timeout from 30s to 60s to leave wider margin for Travis.
+      # increase default timeouts to leave wider margin for Travis.
+      # 30s to 60s
       akka.testconductor.barrier-timeout=60s
-
+      # 5s to 10s
+      lagom.persistence.ask-timeout = 9s
+      # 5s to 10s
+      akka.test.single-expect-default = 11s
+      ## use 9s and 11s for the above timeout because it's coprime values and it'll be easier to spot interferences.
+      ## Also, make the Akka expect() timeouts higher since this tests often expect over an ask operation.
+ |
```

Comment:
@ignasi35, this trailing | is causing the build to fail

Reply:
Bollocks